frontend/src/AddMovie/AddNewMovie/AddNewMovie.js

```diff
@@ -141,6 +143,8 @@ class AddNewMovie extends Component {
                     return (
                       <AddNewMovieSearchResultConnector
                         key={item.tmdbId}
+                        queueItems={queueItems}
```

Comment:
You aren't using this in AddNewMovieSearchResultConnector.js, you are getting queue from store there. I don't think its needed here.